pyfr/solvers/baseadvecdiff/elements.py

```diff
@@ -10,6 +10,9 @@ def _scratch_bufs(self):
         if 'flux' in self.antialias:
             bufs |= {'scal_qpts', 'vect_qpts'}
 
+        if self.basis.fpts_in_upts:
```

Comment:
Is this always what we want?  Are there ever exceptions based on AA and the such like?

Reply:
from my understanding, when we use flux point AA, the flux points 'fpts' are set as the AA quadrature points, and as a result the self.fpts_in_upts will fail. So I think this should be fine.